File: MarketplaceJPA/src/main/java/com/skilldistillery/marketplace/entities/Category.java
package com.skilldistillery.marketplace.entities;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Category {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	private String type;
	
	private String description;
	
	
	
	
	

	public Category() {
		super();
	}

	public Category(int id, String type, String description) {
		super();
		this.id = id;
		this.type = type;
		this.description = description;
	}
	
	

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	@Override
	public String toString() {
		StringBuilder builder = new StringBuilder();
		builder.append("Category [id=").append(id).append(", type=").append(type).append(", description=")
				.append(description).append("]");
		return builder.toString();
	}

}

File: MarketplaceJPA/src/main/java/com/skilldistillery/marketplace/entities/Category.kt
package com.skilldistillery.marketplace.entities

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
class Category {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id = 0
    var type: String? = null
    var description: String? = null

    constructor() : super()
    constructor(id: Int, type: String?, description: String?) : super() {
        this.id = id
        this.type = type
        this.description = description
    }

    override fun toString(): String {
        return "Category [id=" + id + ", type=" + type + ", description=" +
                description + "]"
    }
}
